From: Braving Authors <support@braving.org>
Date: Fri, 5 Sep 2026 10:15:00 +0000
Subject: [PATCH] [branding] Rebrand application name and package ID to Braving

This patch changes the default Android package manifest name from
com.brave.browser to org.braving.browser (and org.braving.browser.debug
for debug builds). It also updates the launcher app strings from Brave
to Braving across Android channel constants.
---
 app/theme/brave/android/res_brave_base/values/channel_constants.xml         | 8 ++++----
 app/theme/brave/android/res_brave_default_base/values/channel_constants.xml | 8 ++++----
 app/theme/brave/android/values/channel_constants.xml                        | 8 ++++----
 build/commands/lib/buildArgs.ts                                             | 10 +++++-----
 4 files changed, 17 insertions(+), 17 deletions(-)

diff --git a/app/theme/brave/android/res_brave_base/values/channel_constants.xml b/app/theme/brave/android/res_brave_base/values/channel_constants.xml
index 4759d6a..8f21975 100644
--- a/app/theme/brave/android/res_brave_base/values/channel_constants.xml
+++ b/app/theme/brave/android/res_brave_base/values/channel_constants.xml
@@ -6,8 +6,8 @@
 
 <resources xmlns:android="http://schemas.android.com/apk/res/android">
     <!-- The application name displayed to the user. -->
-    <string name="app_name" translatable="false">Brave</string>
-    <string name="bookmark_widget_title" translatable="false">Brave bookmarks</string>
-    <string name="search_widget_title" translatable="false">Brave search</string>
-    <string name="quick_action_search_widget_title" translatable="false">Brave quick action search</string>
+    <string name="app_name" translatable="false">Braving</string>
+    <string name="bookmark_widget_title" translatable="false">Braving bookmarks</string>
+    <string name="search_widget_title" translatable="false">Braving search</string>
+    <string name="quick_action_search_widget_title" translatable="false">Braving quick action search</string>
 </resources>
diff --git a/app/theme/brave/android/res_brave_default_base/values/channel_constants.xml b/app/theme/brave/android/res_brave_default_base/values/channel_constants.xml
index 9e9a4f6..178229b 100644
--- a/app/theme/brave/android/res_brave_default_base/values/channel_constants.xml
+++ b/app/theme/brave/android/res_brave_default_base/values/channel_constants.xml
@@ -6,8 +6,8 @@
 
 <resources xmlns:android="http://schemas.android.com/apk/res/android">
     <!-- The application name displayed to the user. -->
-    <string name="app_name" translatable="false">Brave - Debug</string>
-    <string name="bookmark_widget_title" translatable="false">Brave bookmarks</string>
-    <string name="search_widget_title" translatable="false">Brave search</string>
-    <string name="quick_action_search_widget_title" translatable="false">Brave quick action search</string>
+    <string name="app_name" translatable="false">Braving - Debug</string>
+    <string name="bookmark_widget_title" translatable="false">Braving bookmarks</string>
+    <string name="search_widget_title" translatable="false">Braving search</string>
+    <string name="quick_action_search_widget_title" translatable="false">Braving quick action search</string>
 </resources>
diff --git a/app/theme/brave/android/values/channel_constants.xml b/app/theme/brave/android/values/channel_constants.xml
index 9e9a4f6..178229b 100644
--- a/app/theme/brave/android/values/channel_constants.xml
+++ b/app/theme/brave/android/values/channel_constants.xml
@@ -6,8 +6,8 @@
 
 <resources xmlns:android="http://schemas.android.com/apk/res/android">
     <!-- The application name displayed to the user. -->
-    <string name="app_name" translatable="false">Brave - Debug</string>
-    <string name="bookmark_widget_title" translatable="false">Brave bookmarks</string>
-    <string name="search_widget_title" translatable="false">Brave search</string>
-    <string name="quick_action_search_widget_title" translatable="false">Brave quick action search</string>
+    <string name="app_name" translatable="false">Braving - Debug</string>
+    <string name="bookmark_widget_title" translatable="false">Braving bookmarks</string>
+    <string name="search_widget_title" translatable="false">Braving search</string>
+    <string name="quick_action_search_widget_title" translatable="false">Braving quick action search</string>
 </resources>
diff --git a/build/commands/lib/buildArgs.ts b/build/commands/lib/buildArgs.ts
index b0fe5f1..92518e9 100644
--- a/build/commands/lib/buildArgs.ts
+++ b/build/commands/lib/buildArgs.ts
@@ -248,15 +248,15 @@ export function getBuildArgs(config: Config) {
     args.android_channel = config.channel
     if (!config.isReleaseBuild()) {
       args.android_channel = 'default'
-      args.chrome_public_manifest_package = 'com.brave.browser_default'
+      args.chrome_public_manifest_package = 'org.braving.browser.debug'
     } else if (config.channel === '') {
       args.android_channel = 'stable'
-      args.chrome_public_manifest_package = 'com.brave.browser'
+      args.chrome_public_manifest_package = 'org.braving.browser'
     } else if (config.channel === 'beta') {
-      args.chrome_public_manifest_package = 'com.brave.browser_beta'
+      args.chrome_public_manifest_package = 'org.braving.browser.beta'
     } else if (config.channel === 'dev') {
-      args.chrome_public_manifest_package = 'com.brave.browser_dev'
+      args.chrome_public_manifest_package = 'org.braving.browser.dev'
     } else if (config.channel === 'nightly') {
       args.android_channel = 'canary'
-      args.chrome_public_manifest_package = 'com.brave.browser_nightly'
+      args.chrome_public_manifest_package = 'org.braving.browser.nightly'
     }
